packages/logo/composer.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "automattic/jetpack-logo",
```

Comment:
I think we need to add a version here for composer to install it until we have the packagist with a tag setup. I think `composer install` is failing for me because there's not a v1.

Once we're setup with packagist, it will automatically handling versioning based on the git tag.

Reply:
Disregard I think. Picked up the last rebase and seems to work now.